cmplibrary/src/main/res/raw/js_receiver.js

```diff
@@ -2,18 +2,25 @@ function isFromPM(payload) {
     return payload.fromPM || (payload.settings && payload.settings.vendorList != null);
 }
 
-function getPmIdFromURL(url) {
-    return url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
+function getPmIdAndPmTabFromURL(url) {
+    let pmId = url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
```

Comment:
avoid using ES6 features like `let` for ensuring cross webview/browser compatibility: use `var` instead

Reply:
Good suggestion, Done.